codalab/worker/docker_utils.py

```diff
@@ -258,10 +258,14 @@ def get_container_running_time(container):
     start_time = container.attrs['State']['StartedAt']
     # Calculate the end_time of the current container. If 'Status' of the current container is not 'exited',
     # then using the current time as end_time
+    # Because naive datetime objects are treated by many datetime methods as local times,
+    # it is preferred to use aware datetimes to represent times in UTC. As such, the
+    # recommended way to create an object representing the current time in UTC is
+    # by calling datetime.now(timezone.utc).
     end_time = (
         container.attrs['State']['FinishedAt']
         if container.attrs['State']['Status'] == 'exited'
-        else str(datetime.datetime.now().replace(tzinfo=tz.tzutc()))
+        else str(datetime.now(timezone.utc))
```

Comment:
It's better to not change docker container running time calculation to `timezone.utz`, as the start time received from Docker rest API is using a special format which sets the `tzinfo`, see the discussion here [PR #1687 ](https://github.com/codalab/codalab-worksheets/pull/1687#discussion_r350552900)

Reply:
Thanks for the reminder!